### PR TITLE
Adds a pausable revokeOperator method.  NO-1096

### PR DESCRIPTION
**Pull request by scottynomad**

### CODE REVIEW

File: contracts/LockedNORI.sol [hunk 1]
```diff
@@ -138,7 +138,7 @@ contract LockedNORI is
     uint256 amount,
     bytes calldata userData,
     bytes calldata operatorData
-  ) external override {
+  ) external override whenNotPaused {
```
Comment (jaycenhorton) [inline contracts/LockedNORI.sol:141]: this is redundant since `_beforeTokenTransfer` uses `whenNotPaused`

Comment (scottynomad) [reply contracts/LockedNORI.sol:141]: Hmm, yeah you're right.  I was thinking there was a path through _depositFor that would still mutate state and create a schedule but as long as it also sent tokens the `_beforeTokenTransfer` hook would be called.

File: contracts/LockedNORI.sol [hunk 2]
```diff
@@ -675,6 +675,16 @@ contract LockedNORI is
     return ERC777Upgradeable.authorizeOperator(operator);
   }
 
+  /**
+   * @dev This is a wrapper, minting is only supported via `send`
+   *
+   * See {ERC20-_mint}.
+   *
+   */
+  function mint(address, uint256) public virtual override {
```
Comment (jaycenhorton) [inline contracts/LockedNORI.sol:684]: I believe with my latest this is also redundant as the public `mint` function is now longer visible by default and so this just exposes it for no reason

(mint was previously public when we were inheriting from the ERC20 preset)

File: contracts/LockedNORI.sol [hunk 3]
```diff
@@ -734,4 +744,30 @@ contract LockedNORI is
...
+   *
+   * Emits {Burned} and {IERC20-Transfer} events.
+   */
+  function operatorBurn(
+    address,
+    uint256,
+    bytes memory,
+    bytes memory
+  ) public virtual override {
+    revert();
```
Comment (jaycenhorton) [inline contracts/LockedNORI.sol:759]: I think the base branch has a better variant of this override with a revert message

File: contracts/LockedNORI.sol [hunk 4]
```diff
@@ -734,4 +744,30 @@ contract LockedNORI is
...
+  }
+
+  /**
+   * @dev See {IERC777-revokeOperator}.
+   */
+  function revokeOperator(address operator)
```
Comment (jaycenhorton) [inline contracts/LockedNORI.sol:765]: this seems to definitely be missing in the base though, so might be the only thing to merge from this branch

**Pull request closed without merging**